README.md

```diff
@@ -46,10 +47,37 @@ The file structure expected for your application or library.
 ```
 
 
+## Config file
+
+* Systematic requires a `systematic.ini` configuration file in the root folder of your project. Find out all the available options in `[systematic.example.ini](blob/master/systematic.example.ini)`
```

Comment:
the systematic.example.ini is outdated :'( 

and it makes a double with the one in the sample app.

I'd better be for removing it and adding a section in the README about all the available options.